platform/darwin/src/MGLPointCollection.h

```diff
@@ -0,0 +1,25 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+#import "MGLShape.h"
+
+@interface MGLPointCollection : MGLShape
```

Comment:
Should this class conform to MGLOverlay? (Intriguingly, we could have this class be a subclass of MGLMultiPoint, in order to share mutable coordinate code. But best to leave this as is for now.)